docs/index.rst

```diff
@@ -1,12 +1,23 @@
-Welcome to pynsot's documentation!
-==================================
-
+Pynsot - The Network Source of Truth Client
+===========================================
 
 Pynsot is the official client library and command-line utility for the
 :abbr:`NSoT (Network Source of Truth)` IPAM.  For more information on the core
```

Comment:
Can you make this a link instead and have it actually spell out Network Source of Truth (since that's the first mention of it?)